### PR TITLE
Remove support for Hexagon v60 architecture.

### PR DESCRIPTION
**Pull request by dsharletg**
This is a pretty old architecture at this point, and the last known use of it has been removed. This PR changes the default baseline architecture to v62.

### CODE REVIEW

**Comment by steven-johnson** [comment]
LGTM

**Comment by dsharletg** [comment]
If you have a better patch, please feel free to open a PR and we can close this one.

**Comment by steven-johnson** [comment]
Where do we stand on this? Are we landing this or is there a 'better' option we are waiting for?

**Comment by dsharletg** [comment]
Just addressed @pranavb-ca's feedback.

File: src/runtime/hexagon_remote/bin/.clang-format [hunk 1]
```diff
@@ -1 +0,0 @@
-DisableFormat: true
```
Comment (steven-johnson) [inline src/runtime/hexagon_remote/bin/.clang-format]: Please undelete this file: we don't want clang-format to be applied to the stuff in hexagon_remote/src

**Comment by dsharletg** [comment]
This needs https://github.com/halide/build_bot/pull/48 to pass the build bots. Should we just submit these at close to the same time? Or is there a better way that I'm missing?

**Comment by steven-johnson** [comment]
submit at close to same time is good. Merge this now, I will get the buildbots updated within a couple of hours.

**Comment by steven-johnson** [comment]
(clang-format nonsense pulled out into https://github.com/halide/Halide/pull/4511 separately)

**Comment by dsharletg** [comment]
The failures look unrelated, the affected tests are passing. I'm going to merge this.

**Pull request merged**